README.md

```diff
@@ -46,6 +46,14 @@ The following dependencies need to be installed on your system to be able to ins
 
 > ✨ Currently, we only support Linux execution. Tested on CentOS 7.5, with miniconda 4.7.12 and Singularity 3.5.2.
 
+### Minimum resources:
+
+A real e.g human CLIP experiment would require:
+- ~40 GB of RAM (mapping of human genome and building genome index relies on this requirement)
+- 100 GB of disk space
```

Comment:
After downloading the test data I get with `du -BM input_files*`
```
33M     input_files/ATtRACT
5249M   input_files
2881M   input_files_test
```
so approx. 10GB of disk space if for the input files.

Reply:
The output (per default the directory `results`) takes 45GB of disk space. Where the STAR index weighs in 28GB.
```tsv
1310M   results/GN
2954M   results/TR
28102M  results/STAR_index
6409M   results/TR_salmon_index
5832M   results/cutadapt
542M    results/ensembl_csv
45148M  results/
```